kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go

```diff
@@ -146,12 +146,12 @@ type SelinuxConfig struct {
 	ChconCmds    [][]string `json:"chconCmds,omitempty"`
 }
 
-type IPTablesConfig struct {
+type IptablesConfig struct {
 	IptablesCmds [][]string `json:"iptablesCmds,omitempty"`
 }
 
 type FirewalldConfig struct {
-	Enabled       bool       `json:"enabled,omitempty"`
+	Firewalld     string     `json:"firewalld,omitempty"`
```

Comment:
Changing because with bool it's impossible to know when it wasn't set (i.e. don't change state).